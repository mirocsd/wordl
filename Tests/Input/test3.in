bacteria
describe
equation
fracture
ceremony
songbird
