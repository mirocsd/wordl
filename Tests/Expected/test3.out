Enter guess: [33mb[0macte[33mri[0ma
Enter guess: [33md[0me[33ms[0mc[33mr[32mi[33mb[0me
Enter guess: equat[32mi[33mon
[0mEnter guess: fractu[32mr[0me
Enter guess: ce[33mr[0mem[33mon[0my
Enter guess: [32msongbird
[0mFinished in 6 guesses
